test/fixtures.py

```diff
@@ -164,7 +186,6 @@ def open(self):
         else:
             raise Exception('Failed to start Zookeeper before max_timeout')
         self.out("Done!")
-        atexit.register(self.close)
```

Comment:
Why is it safe to remove this?

Reply:
Because I'm handling the destruction at the fixture method:

```
+def zookeeper():
     zk = ZookeeperFixture.instance()
     yield zk
     zk.close() # <--
```